dds_web/database/models.py

```diff
@@ -130,7 +122,42 @@ class User(db.Model):
     def __repr__(self):
         """Called by print, creates representation of object"""
 
-        return f"<User {self.username}>"
+        return f"<User {self.username}:{self.role}>"
```

Comment:
The base User class does not have a role column. How does this work?

Reply:
It would indeed crash for a object which is of the class User but not of any of the subclasses UnitUser or ResearchUser. We should probably define that property in the base class as well and set it to None or something.

Reply:
+1